app/controllers/ApplicationController.scala

```diff
@@ -19,19 +19,13 @@ package controllers
 import audit.Auditable
 import config.ApplicationConfig
 import javax.inject.Inject
-import models.FeedBack
-import models.FeedbackForm.feedbackForm
 import play.api.i18n.I18nSupport
 import play.api.mvc.{Action, AnyContent, DiscardingCookie, MessagesControllerComponents}
-import uk.gov.hmrc.http.HeaderCarrier
-import uk.gov.hmrc.play.audit.model.EventTypes
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
-import utils.RedirectUtils
 
 class ApplicationController @Inject()(val config: ApplicationConfig,
                                       audit: Auditable,
                                       templateUnauthorised: views.html.unauthorised,
-                                      templateFeedBack: views.html.feedback,
                                       templateThankYou: views.html.feedbackThankYou,
```

Comment:
This looks as though it should also be removed as its part of the same process for feedback

Reply:
templateFeedBack and templateThankYou removed